numba_cuda/numba/cuda/cudadrv/nvrtc.py

```diff
@@ -237,7 +237,7 @@ def compile(src, name, cc):
 
     cudadrv_path = os.path.dirname(os.path.abspath(__file__))
     numba_cuda_path = os.path.dirname(cudadrv_path)
-    numba_include = f'-I{numba_cuda_path}'
+    numba_include = f'-I{numba_cuda_path} -I{_get_include_dir()}'
```

Comment:
It looks like it's a bug here to return the result of `_get_include_dir()`. Because it returns a named tuple, but `numba_include` takes the path to the include dir.

Secondly, the include path seems to conflict with the `include` flag that's already passing into nvrtc. There should be a single source of CTK include, whether its site installation or conda.

My experiment seems to show that for conda installed includes, a more robust path is `$CONDA_PREFIX/targets/<this_machine_arch>/include`.

Finally, cmake uses a different way to find CTK at compile time, which greps output from nvcc compiling an empty file. Adopting this to Numba would be a great change and is certainly out of scope here - but would it be a correct move forward?